[2025-03-21T17:42:32.569+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-21T17:42:32.579+0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_twitter.fetch_twitter_data_task manual__2025-03-21T12:36:46.096700+00:00 [queued]>
[2025-03-21T17:42:32.583+0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_twitter.fetch_twitter_data_task manual__2025-03-21T12:36:46.096700+00:00 [queued]>
[2025-03-21T17:42:32.584+0500] {taskinstance.py:2867} INFO - Starting attempt 2 of 4
[2025-03-21T17:42:32.598+0500] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_twitter_data_task> on 2025-03-21 12:36:46.096700+00:00
[2025-03-21T17:42:32.603+0500] {standard_task_runner.py:72} INFO - Started process 45692 to run task
[2025-03-21T17:42:32.606+0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_twitter', 'fetch_twitter_data_task', 'manual__2025-03-21T12:36:46.096700+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/fetch_twitter.py', '--cfg-path', '/tmp/tmp78s65u6w']
[2025-03-21T17:42:32.607+0500] {standard_task_runner.py:105} INFO - Job 18: Subtask fetch_twitter_data_task
[2025-03-21T17:42:32.641+0500] {task_command.py:467} INFO - Running <TaskInstance: fetch_twitter.fetch_twitter_data_task manual__2025-03-21T12:36:46.096700+00:00 [running]> on host ubuntu
[2025-03-21T17:42:32.690+0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fetch_twitter' AIRFLOW_CTX_TASK_ID='fetch_twitter_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T12:36:46.096700+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-21T12:36:46.096700+00:00'
[2025-03-21T17:42:32.691+0500] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-21T17:42:32.691+0500] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-21T17:42:32.691+0500] {logging_mixin.py:190} INFO - Current task name:fetch_twitter_data_task state:running start_date:2025-03-21 12:42:32.579511+00:00
[2025-03-21T17:42:32.691+0500] {logging_mixin.py:190} INFO - Dag name:fetch_twitter and current dag run status:running
[2025-03-21T17:42:32.692+0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-21T17:42:33.109+0500] {logging_mixin.py:190} INFO - Error fetching tweets: 403 Forbidden
453 - You currently have access to a subset of X API V2 endpoints and limited v1.1 endpoints (e.g. media post, oauth) only. If you need access to this endpoint, you may need a different access level. You can learn more here: https://developer.x.com/en/portal/product
[2025-03-21T17:42:33.110+0500] {python.py:240} INFO - Done. Returned value was: None
[2025-03-21T17:42:33.114+0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-21T17:42:33.115+0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fetch_twitter, task_id=fetch_twitter_data_task, run_id=manual__2025-03-21T12:36:46.096700+00:00, execution_date=20250321T123646, start_date=20250321T124232, end_date=20250321T124233
[2025-03-21T17:42:33.137+0500] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-21T17:42:33.138+0500] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-21T17:42:33.138+0500] {logging_mixin.py:190} INFO - Dag name:fetch_twitter queued_at:2025-03-21 12:36:46.109433+00:00
[2025-03-21T17:42:33.138+0500] {logging_mixin.py:190} INFO - Task hostname:ubuntu operator:PythonOperator
[2025-03-21T17:42:33.181+0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-21T17:42:33.214+0500] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-21T17:42:33.215+0500] {local_task_job_runner.py:245} INFO - ::endgroup::
